chart/skywalking/values.yaml

```diff
@@ -451,6 +451,7 @@ satellite:
   ports:
     grpc: 11800
     prometheus: 1234
+    pprof: 6060
```

Comment:
Do we need to enable this by default? This is for debugging no?

Reply:
I think it can be added for observing and querying execution status. If you think it's not necessary to turn it on, I can close this PR.